ext_emconf.php

```diff
@@ -12,11 +12,7 @@
   'author_email' => 'support@mittwald.de',
   'author_company' => 'Mittwald CM Service GmbH',
   'state' => 'beta',
-<<<<<<< HEAD
-  'version' => '1.1.0',
-=======
-  'version' => '1.0.0',
->>>>>>> parent of 812eb1f... v1.10
+  'version' => '1.1.1',
```

Comment:
Also, there should also be a reference to the library in the `composer.json` file. Please change that also (or add, if not already present).